docs/src/Datapoint.doc.js

```diff
@@ -0,0 +1,140 @@
+// @flow strict
+import React, { type Node } from 'react';
+import { Datapoint } from 'gestalt';
+import PropTable from './components/PropTable.js';
+import PageHeader from './components/PageHeader.js';
+import MainSection from './components/MainSection.js';
+import FeedbackCallout from './components/FeedbackCallout.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(<FeedbackCallout componentName="Datapoint" />);
+
+card(
+  <PageHeader
+    name="Datapoint"
+    description="Datapoint displays at-a-glance data for a user to quickly view key metrics."
+    defaultCode={`
+<Datapoint size="lg" infoText="Contextual information for this Datapoint" title="Datapoint title" value="1.23M" trend={{value: 30, accessibilityLabel: "Trending up"}} helperText="Datapoint helper text" />
+`}
+    pilot
+  />,
+);
+
+// Specifying Component gives auto warnings about missing props
+card(
+  <PropTable
+    Component={Datapoint}
+    props={[
+      {
+        name: 'title',
+        type: 'string',
+        description: 'The header for the component',
+        required: true,
+      },
+      {
+        name: 'value',
+        type: 'string',
+        description: 'The main datapoint value (e.g., 1.23M)',
+        required: true,
+      },
+      {
+        name: 'trend',
+        type: '{| accessibilityLabel: string, value: string |}',
```

Comment:
If we're doing comparisons to 0, the value needs to be a number, not a string. We don't want to rely on parsing a string as that's brittle (and isn't currently happening in this PR)

Reply:
Fixed - thanks!